src/details/ArborX_DetailsDistributedTreeImpl.hpp

```diff
@@ -32,8 +32,25 @@
 namespace ArborX
 {
 
+struct PairIndexRank
+{
+  int index;
+  int rank;
+  friend bool operator==(PairIndexRank lhs, PairIndexRank rhs)
```

Comment:
```suggestion

private:
  friend bool operator==(PairIndexRank lhs, PairIndexRank rhs)
```
Where are these needed?  Tests?
Should they be annotated with KOKKOS_FUNCTION?
Should they be constexpr?

Reply:
Ping

Reply:
Seems like is only needed in testing (`tstDistributedTree.cpp`). Moved it there. The annotation happened in @dalg24's patch.